testing.java

```diff
@@ -1,3 +1,3 @@
 public class testing {
-    //hello world this is test branch
+    //hello world this is a conflict testing branch
```

Comment:
can change the spelling of world to WORLD and hello to HELLO

Reply:
done